app/src/main/java/com/example/henzoshimada/feeltrip/FeelTripApplication.java

```diff
@@ -59,4 +65,79 @@ public static Participant getParticipant(){
         return participant;
     }
 
+    private static FilterController filterController = null;
+    public static FilterController getFilterController(){
+        if (filterController == null){
+            filterController = new FilterController();
+        }
+        return filterController;
+    }
+
+    private static  MoodAdapter moodAdapter = null;
+    private static ArrayList<Mood> moodArrayList = new ArrayList<Mood>();
+    public static MoodAdapter getMoodAdapter(Context context) {
+        if (moodAdapter == null) {
+            moodAdapter = new MoodAdapter(moodArrayList, context.getApplicationContext());
+        }
+        return moodAdapter;
```

Comment:
For the purposes of refreshing the MoodAdapter no matter where we are in the app.